src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Jeroen Thora <jeroen.thora@intracto.com>
+ */
+class ProvinceApiTest extends JsonApiTestCase
+{
+    /**
+     * @var array
+     */
+    private static $authorizedHeaderWithContentType = [
+        'HTTP_Authorization' => 'Bearer SampleTokenNjZkNjY2MDEwMTAzMDkxMGE0OTlhYzU3NzYyMTE0ZGQ3ODcyMDAwM2EwMDZjNDI5NDlhMDdlMQ',
+        'CONTENT_TYPE' => 'application/json',
+    ];
+
+    public function testGetProvinceAccessDeniedResponse()
```

Comment:
Yes we weren't sure what the convention was, but as more tests had camelcase function names we assumed that camelcase would be the standard